Topics/Development_Process/App_Store_Deployment.md

```diff
@@ -0,0 +1,144 @@
+# Apple App Store Deployment
+
+## Introduction
+After working towards a useable version of your mobile app, your last step would be to handle the deployment side. 
+The main difficulties faced with submitting an app to the App Store is that they set high standards for privacy and security. 
+You would need various certificates in order to distribute your app to the App Store as well as use certain features like: Push Notifications, Apple Pay, etc.. 
+These are some possible hurdles to keep in mind when working on a project to release an iOS app, but they certainly help with raising the quality of your project.
+
+## Requirements
+- Apple Developer Account
+- Apple Certificates
+- Xcode (Requires a MacBook)
+- Established Privacy Policy & Terms and Conditions
+
+
+### Apple Developer Program
+The most crucial component you will need to be able to deploy to the App Store is to enroll in the Apple Developer Program. 
+You may enroll as an individual or as an organization, but keep in mind that they have different requirements for each option.
+
+
+**EXTRA REQUIREMENTS**:
```

Comment:
Should put every requirement in the same section, you can specify there that these are more specific / extra.

Reply:
okay, sounds good